src/app/shared/cookies/browser-klaro.service.spec.ts

```diff
@@ -233,4 +257,50 @@ describe('BrowserKlaroService', () => {
       expect(ePersonService.patch).not.toHaveBeenCalled();
     });
   });
+
+  describe('initialize google analytics configuration', () => {
+    let GOOGLE_ANALYTICS_KEY;
+    beforeEach(() => {
+      GOOGLE_ANALYTICS_KEY = clone((service as any).GOOGLE_ANALYTICS_KEY);
+      configurationDataService.findByPropertyName = findByPropertyName;
+      spyOn((service as any), 'getUser$').and.returnValue(observableOf(user));
+      translateService.get.and.returnValue(observableOf('loading...'));
+      spyOn(service, 'addAppMessages');
+      spyOn((service as any), 'initializeUser');
+      spyOn(service, 'translateConfiguration');
+    });
+    it('should not filter any configuration', () => {
```

Comment:
Super minor thing; would be good to specify here that this is the case when the GA configuration does exist on REST, otherwise you have to keep in mind how you set up the spy above. In smaller test suites it's not a big issue but it's a bit confusing IMO.

Reply:
Done, Thank You @ybnd